src/ziggurat/init.clj

```diff
@@ -247,6 +253,7 @@
    (try
      (let [derived-modes (validate-modes modes stream-routes batch-routes actor-routes)]
        (initialize-config)
+       (set-properties-for-structured-logging)
```

Comment:
should be part of `initialize-config` rather than being a function here in this namespace.

Reply:
addressed it.